lang/en.yml

```diff
@@ -12,16 +12,16 @@ en:
     db_RealmeSPNameID: 'RealmeSP name ID'
   SilverStripe\RealMe\RealMeService:
     ERROR_AUTHNFAILED: 'You have chosen to leave RealMe.'
-    ERROR_GENERAL: 'RealMe reported a serious application error. Please try again later. If the problem persists, please contact the RealMe Help Desk. From New Zealand: 0800 664 774 (toll free), from overseas dial +64 9 357 4468 (overseas call charges apply).'
-    ERROR_INTERNAL: 'RealMe was unable to process your request due to a RealMe internal error. Please try again. If the problem persists, please contact the RealMe Help Desk. From New Zealand dial 0800 664 774 (toll free), from overseas dial +64 9 357 4468 (overseas call charges apply).'
-    ERROR_NOAUTHNCONTEXT: "RealMe reported a serious application error with the message 'No AuthN Context'. Please try again later. If the problem persists, please contact the RealMe Help Desk. From New Zealand: 0800 664 774 (toll free), from overseas dial +64 9 357 4468 (overseas call charges apply)."
-    ERROR_NOAVAILABLEIDP: 'RealMe reported that the TXT service or the token service is not available. You may try again later. If the problem persists, please contact the RealMe Help Desk. From New Zealand dial 0800 664 774 (toll free), from overseas dial +64 9 357 4468 (overseas call charges apply).'
-    ERROR_NOPASSIVE: "RealMe reported a serious application error with the message 'No Passive'. Please try again later. If the problem persists, please contact the RealMe Help Desk. From New Zealand: 0800 664 774 (toll free), from overseas dial +64 9 357 4468 (overseas call charges apply)."
-    ERROR_REQUESTDENIED: "RealMe reported a serious application error with the message 'Request Denied'. Please try again later. If the problem persists, please contact the RealMe Help Desk. From New Zealand: 0800 664 774 (toll free), from overseas dial +64 9 357 4468 (overseas call charges apply)."
-    ERROR_REQUESTUNSUPPORTED: "RealMe reported a serious application error with the message 'Request Unsupported'. Please try again later. If the problem persists, please contact the RealMe Help Desk. From New Zealand: 0800 664 774 (toll free), from overseas dial +64 9 357 4468 (overseas call charges apply)."
+    ERROR_GENERAL: 'RealMe reported a serious application error. Please try again later. If the problem persists, please contact the RealMe Help Desk. From New Zealand: 0800 664 774 (toll free), from overseas dial +64 4 462 0674 (overseas call charges apply).'
+    ERROR_INTERNAL: 'RealMe was unable to process your request due to a RealMe internal error. Please try again. If the problem persists, please contact the RealMe Help Desk. From New Zealand dial 0800 664 774 (toll free), from overseas dial +64 4 462 0674 (overseas call charges apply).'
```

Comment:
This phone number is correct according to https://www.realme.govt.nz/contact-us/
Not sure why it's changing, but it _is_ changing to the correct value.